docs/ldml/tr35-general.md

```diff
@@ -948,25 +948,27 @@ Some of the constraints reference data from the unitIdComponents in [Unit_Conver
         <li>See <a href="#Private_Use_Units">Private-Use Units</a></li>
     </ul></td></tr>
 
-<tr><td>number_prefix</td><td>:=</td>
-    <td>("1"[0-9]+ | [2-9][0-9]*) "-"
+<tr><td><a name='unit_constant' href='unit_constant'>unit_constant</a></td><td>:=</td>
+    <td>("1"[0-9]+ | [2-9][0-9]*)("e" ("1"[0-9]+ | [2-9][0-9]*))?
         <ul><li><em>Examples:</em>
             <ul><li>kilowatt-hour-per-100-kilometer</li>
                 <li>gallon-per-100-mile</li>
                 <li>per-200-pound</li>
+                <li>per-12</li>
             </ul></li>
             <li><em>Note:</em> The number is an integer greater than one.</li>
+            <li><em>Note:</em> The <code>e</code> notation is optional: per-100-kilometer and per-1e2-kilometer are equivalent unit_identifiers.</li>
```

Comment:
1e2 or 1e9 does not match “("1"[0-9]+ | [2-9][0-9]*)("e" ("1"[0-9]+ | [2-9][0-9]*))?”

nor 1e2 1e3 1e4 1e5 etc

The part before “e” and after “e” only match “2”, “3”, “4” …. “99” with the definition of ("1"[0-9]+ | [2-9][0-9]*)